src/react/account/details/properties/AccountKeys.jsx

```diff
@@ -0,0 +1,174 @@
+    const dispatch = useDispatch();
+    const accessKeysInfo = useSelector((state: AppState) => state.user.accessKeyList);
+
+    useEffect(() => {
+        dispatch(listAccountAccessKeys(account.id));
+    }, [dispatch, account.id]);
+
+    const columns = useMemo(() => [
+        {
+            Header: 'Access Key',
+            accessor: 'access_key',
+            Cell({ value: access_key }: { value: string }) {
+                return <span style={{ display: 'flex', justifyContent: 'space-between' }}>
+                    <EllipsisCell title={access_key}>{access_key}</EllipsisCell>
+                    <Clipboard text={access_key}/>
+                </span>;
+            },
+        },
+        {
+            Header: 'Created On',
+            accessor: 'created_at',
+            Cell({ value: created_at }: { value: Date }) { return formatDate(new Date(created_at)); },
+        },
+        {
+            id: 'actions',
+            Header: '',
+            accessor: 'access_key',
+            disableSortBy: true,
+            Cell({ value: access_key }: { value: string }){
+                return (
+                    <T.Actions>
+                        <Tooltip overlay='Remove Key' placement='left'>
+                            <T.ActionButton
+                                title='Remove Key'
+                                disabled={false}
+                                icon={<i className='fas fa-trash' />}
+                                onClick={() => dispatch(deleteAccountAccessKey(access_key, account.id))}
+                                size='smaller'
+                                variant='danger'
+                                text=''
+                            />
+                        </Tooltip>
+                    </T.Actions>
+                );
+            },
+        },
+    ], [account.id, dispatch]);
+
+    const accessKeys = useMemo(() => accessKeysInfo.map(accessKeyInfo => {
+        return {
+            access_key: accessKeyInfo.AccessKeyId,
+            created_at: accessKeyInfo.CreateDate,
+        };
+    }), [accessKeysInfo]);
+
+    const {
+        getTableProps,
+        getTableBodyProps,
+        headerGroups,
+        rows,
+        prepareRow,
+    } = useTable({
+        columns,
+        data: accessKeys,
+        initialState: { sortBy: initialSortBy },
+        disableSortRemove: true,
+        autoResetFilters: false,
+        autoResetSortBy: false,
+    }, useFilters, useSortBy);
+
+    return (
+        <TableContainer>
+            <h3>Root user Access Key & Secret Keys details</h3>
+            <div style={{ display: 'flex' }}>
+                { accessKeys && accessKeys.length >= 1 &&
+                    <Banner
+                        variant='danger'
+                        icon={<i className='fas fa-exclamation-triangle' />}
+                    >
+                        Security Status: delete your Access keys (as a best practice, create an IAM user with access keys).
```

Comment:
```suggestion
                        Security Status: root user access keys gives unrestricted
                        access to account resources. It is recommended to delete
                        root access keys and use IAM user access keys.
```
@wabernatScality fell free to suggest a better working.

Reply:
```suggestion
                        Security Status: Root user access keys give unrestricted
                        access to account resources. It is a best practice to delete
                        root access keys and use IAM user access keys instead.
```